### ISSUE TITLE
Comments for "Has Jupiter Broadcasting Gone Down The Pan?"

### ISSUE THREAD

**Issue by kevquirk** [kevquirk/kevq.uk-post-comments#52]

**Comment by jay-tuckey** [comment]
Hi Kev, I suspect that the ACloudGuru ads were agreed apon before Jupiter Broadcasting split from the company, so what I'm hoping is that once that agreement comes to an end the paid feeds they will be actually ad-free.

**Comment by kevquirk** [comment]
> Hi Kev, I suspect that the ACloudGuru ads were agreed apon before Jupiter Broadcasting split from the company, so what I'm hoping is that once that agreement comes to an end the paid feeds they will be actually ad-free.

Yeah, I really hope so. In fairness to them, the ACG ads are generally much shorter than the other adverts. I just think there's a better way for them to do all this, but we obviously don't know if their hands are contractually tied.

**Comment by oscarcarlsson** [comment]
> What do you think? Am I being too harsh?

I'm not a fan of the ads, but I'm not supporting economically either.

However, if you listen carefully to what is said about the 'ad-free' feed is that Chris mentions that it will include some contractually obligated ads. Too bad this feed is described as 'ad-free' but I think they've stopped calling it 'ad-free'.

**Comment by kevquirk** [comment]
> However, if you listen carefully to what is said about the 'ad-free' feed is that Chris mentions that it will include some contractually obligated ads. Too bad this feed is described as 'ad-free' but I think they've stopped calling it 'ad-free'.

That's good. Let's hope things improve if/when they are released from this contract with ACG.
